Client/Frontend/Onboarding/Models/OnboardingCardInfoModel.swift

```diff
@@ -12,6 +12,7 @@ struct OnboardingCardInfoModel: OnboardingCardInfoModelProtocol {
     var buttons: OnboardingButtons
     var type: OnboardingType
     var a11yIdRoot: String
+    var defaultSettingsButton: OnboardingDefaultBrowserModelProtocol?
```

Comment:
This should not be here. I know we've talked about this offline already, so I won't add more details here. :)